apps/ideas/src/components/modules/WorkshoppingPage/HeartButton.js

```diff
@@ -0,0 +1,17 @@
+// test file for the like button on the workshopping page
```

Comment:
When you pick up a new ticket, could you please branch off of the `development/ideaspace` branch, to avoid bringing in changes from other tickets?

Since this ticket is currently blocked, we don't actually want to merge the code changes with the changes for #768